### PR TITLE
update react-lazy-load-image-component: version 1.5.0

### PR DESCRIPTION
**Pull request by kodai3**
added `wrapperProps` property
which was featured on [version 1.5.0 update](https://github.com/Aljullu/react-lazy-load-image-component/releases/tag/1.5.0)

Thanks! 

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Aljullu/react-lazy-load-image-component/pull/70
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by danvk** [comment]
This change looks fine — but most of it is random formatting diffs that don't have anything to do with `wrapperProps`. Unless this is a new DT standard formatting, could you get rid of those? Otherwise looks good to me.

File: types/react-lazy-load-image-component/index.d.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 // Definitions by: Dan Vanderkam <https://github.com/danvk>
 //                 Diego Chavez <https://github.com/diegochavez>
 //                 Truong Hoang Dung <https://github.com/revskill10>
+//                 Kodai Suzuki <https://github.com/kodai3>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.5
 
 import {
-    CSSProperties,
+	CSSProperties,
```
Comment (danger-public) [inline types/react-lazy-load-image-component/index.d.ts:11]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by kodai3** [comment]
thanks for reviewing

bit straggled to resetting my editor, but  removed personal formatting

**Review by danvk — APPROVED** [review]
Thanks @kodai3 — looks great!

**Comment by kodai3** [comment]
Ready to merge

**Pull request merged**